src/main/java/worldofzuul/Rooms/Property.java

```diff
@@ -0,0 +1,39 @@
+package worldofzuul.Rooms;
+
+import worldofzuul.EnergySources.EnergySource;
+import worldofzuul.Game;
+
+import java.sql.SQLOutput;
+import java.util.ArrayList;
+
+
+public class Property extends Room {
+    private ArrayList<EnergySource> energySources = new ArrayList<>();
```

Comment:
We should consider removing `player.energySources` at some point, but that is probably out of scope for this pr.